core/src/main/kotlin/org/evomaster/core/problem/graphql/schema/__EnumValue.kt

```diff
@@ -0,0 +1,12 @@
+package org.evomaster.core.problem.graphql.schema
+
+/**
+ *  Field: __enumValue part of the introspection system.
+ */
+data class __EnumValue (var name:String,
+                        var isDeprecated: Boolean= true,
```

Comment:
any reason for why `isDeprecated` is true by default?

Reply:
Hi, i worked in a different file in intellij (a different project for graphql that i create to work with) so i did modification in this file and not in evomaster. I will copy them now. The same thing for isDeprecated.
 Now i am working directlly on evomaster.